docs/keywords.json

```diff
@@ -4561,5 +4561,17 @@
     ],
     "category":"Charts and Widgets",
     "release":"2.3.5"
-  }
+  },
+  "no-map":{
+    "alias": [
+      "map"",
```

Comment:
extra `"` on this line I think is screwing up the rest of this entry

Reply:
Thanks a lot. I've updated the code. Please review again.